forge-gui/res/adventure/common/custom_cards/sliver_queen_boss_effect.txt

```diff
@@ -0,0 +1,13 @@
+Name:Sliver Queen's Boss Effect
+ManaCost:no cost
+Colors:all
+Types:Enchantment
+A:AB$ Token | Cost$ 4 | TokenAmount$ 1 | TokenScript$ c_1_1_sliver | ActivationZone$ Command | TokenOwner$ You | Activator$ Player | ActivationLimit$ 1 | SpellDescription$ Create a 1/1 colorless Sliver creature token. Every player may activate this ability but only once each turn.
+T:Mode$ Phase | Phase$ Upkeep | ValidPlayer$ You | TriggerZones$ Command | Execute$ TrigCharm | TriggerDescription$ At the beginning of your upkeep, ABILITY.
+SVar:TrigCharm:DB$ Charm | Random$ True | Choices$ DBToken,DBChoose,DBSeek
+SVar:DBToken:DB$ Token | TokenAmount$ 1 | TokenScript$ c_1_1_sliver | SpellDescription$ Create a 1/1 colorless Sliver creature token.
+SVar:DBChoose:DB$ ChooseCard | AtRandom$ True | Choices$ Creature.nonToken+OppCtrl | SubAbility$ DBCopy | SpellDescription$ Create a token of a random nontoken creature your opponent controls. That creature becomes a Sliver in addition to it's other types.
+SVar:DBCopy:DB$ CopyPermanent | Defined$ ChosenCard | AddTypes$ Sliver | SubAbility$ DBCleanup
+SVar:DBCleanup:DB$ Cleanup | ClearChosenCard$ True
+SVar:DBSeek:DB$ Seek | Type$ Card.Sliver | SpellDescription$ Seek a Sliver card. 
+Oracle:{4}: Create a 1/1 colorless Sliver creature token. Every player may activate this ability but only once each turn. \n At the gebinning of your upkeep, choose one at random\n• Create a 1/1 colorless Sliver creature token.\nCreate a token of a random nontoken creature your opponent controls. That creature becomes a Sliver in addition to it's other types.\n• Seek a Sliver card.
```

Comment:
A mistype: should be "...in addition to its other types."  (+ the same on the picture, if possible)

Reply:
Thanks, fixed